Cubical/Foundations/HLevels.agda

```diff
@@ -290,6 +243,54 @@ isContrRetractOfConstFun : {A : Type ℓ} {B : Type ℓ'} (b₀ : B)
 fst (isContrRetractOfConstFun b₀ ret) = ret .fst b₀
 snd (isContrRetractOfConstFun b₀ ret) y = ret .snd y
 
+-- h-level of dependent path types
+
+isOfHLevelPathP' : {A : I → Type ℓ} (n : HLevel)
+                   → isOfHLevel (suc n) (A i1)
+                   → (x : A i0) (y : A i1) → isOfHLevel n (PathP A x y)
+isOfHLevelPathP' {A = A} n h x y =
+  isOfHLevelRetractFromIso n (PathPIsoPath _ x y) (isOfHLevelPath' n h _ _)
+
+isOfHLevelPathP : {A : I → Type ℓ} (n : HLevel)
+                  → isOfHLevel n (A i1)
+                  → (x : A i0) (y : A i1) → isOfHLevel n (PathP A x y)
+isOfHLevelPathP {A = A} n h x y =
+  isOfHLevelRetractFromIso n (PathPIsoPath _ x y) (isOfHLevelPath n h _ _)
+
+-- Fillers for cubes from h-level
+
+isSet→isSet' : isSet A → isSet' A
+isSet→isSet' {A = A} Aset a₀₋ a₁₋ a₋₀ a₋₁ =
+  transport⁻ (PathP≡Path (λ i → a₋₀ i ≡ a₋₁ i) a₀₋ a₁₋) (Aset _ _ _ _)
```

Comment:
Why not prove this as

```
isSet→isSet' : isSet A → isSet' A
isSet→isSet' {A = A} Aset a₀₋ a₁₋ a₋₀ a₋₁ = toPathP (Aset _ _ _ _)
```
?

Same for other things in this file. It seems really dangerous to transport in this complicated family...

Reply:
Btw, the `PathP≡Path` used here is not `ua` of the proof you simplified in Foundations.Path in this PR

Reply:
Oops, I thought I got them all. I don't see any other uses of `PathP≡Path` I missed though.
I think it's best to leave `PathP≡Path` as it is instead of defining it with `ua`... if one really needs the path, then probably the version that avoids `Glue` is better.